docs/_snippets/features/wproofreader.js

```diff
@@ -78,3 +78,14 @@ ClassicEditor
 	.catch( err => {
 		console.error( err.stack );
 	} );
+
+// external source exclusion
```

Comment:
Or even more meaningful:

```suggestion
// Ignore failures from the `webspellchecker.net` service due to random crawler failures.
```